### ISSUE TITLE
Code Blocks in Graph View sometimes don't properly expand

### ISSUE THREAD

**Issue by ebfe2326** [NationalSecurityAgency/ghidra#1008]
**Describe the bug**
Some of the code blocks in the graph view don't properly expand to show the full width.  I've used the "Edit Code Block Features" button to expand the field widths.  This works on 95% of the code blocks and I can set the field size to see the entire instructions.  For maybe 5% of the code blocks, the fields do not expand like the other 95% and the fields are truncated.

For some reason, this seems to virtually always occur in the microsoft compiler's optimization for mod by a power of 2.  I put two different screenshots of this to highlight, but i've noticed the truncation in every instance where i've seen the optimized mod by power of 2.  But it does occur in other instances too.  The 3rd example is just a random call where the field doesn't get expanded for some reason.  

**Expected behavior**
I would expect all fields to expand to the size set in the "Edit Code Block Features" setting.

**Screenshots**
Example 1 showing the microsoft compiler optimization for mod by power of 2 with the code block on the right side of the screen not properly expanding.  Note the middle Or instruction has an immediate value truncated from the screen.

![image](https://user-images.githubusercontent.com/44279601/64907524-bb3ed480-d6c1-11e9-9a64-9985ae33425c.png)

Example 2 showing the same compiler optimization for Mod by a power of 2 as example 1, but a different location, also not expanding the code block properly.  Note the middle Or instruction has an immediate value truncated from the screen.

![image](https://user-images.githubusercontent.com/44279601/64907551-05c05100-d6c2-11e9-81ff-1b893db6bafd.png)

Example 3 showing a random code block at top of screen not properly expanding
![image](https://user-images.githubusercontent.com/44279601/64907580-7b2c2180-d6c2-11e9-92b5-571d6a724e6a.png)

**Environment (please complete the following information):**
 - OS: [e.g. macOS 10.14.2]
 - Java Version: [e.g. 11.0]
 - Ghidra Version: [e.g. 9.0]

**Comment by dragonmacher** [comment]
I have seen similar behavior after changing the format of the code blocks.  For me, performing a relayout of the graph caused the block to resize correctly.

If you reload the graph by pressing the refresh button does the problem go away?

**Comment by ebfe2326** [comment]
I tried performing a relayout of the graph and also changing the "use condensed view" option.  In both cases, the code block is still truncated.

I did a little more checking though and it looks like in this edge case, adjusting the code block field sizes is making a difference.  If I used the default code browser tool where most of the instructions are truncated, this particular code block doesn't get truncated.  If I expand the address field so I can see the full address instead of only the last 4 digits, most of the code blocks readjust ok but this situation the operands shrink as the address field gets bigger.  After making the change, I tried performing a relayout but it didn't fix it.  When I adjust the different field sizes, 95% of the code blocks adjust properly.  For some reason, these 5% don't seem to be adjusting.  I've definitely seen this situation in different code block types, but i always see it during the microsoft compiler mod by power of 2 optimization that I put the screenshots for.  It's seems really strange that it's only happening in a small percentage of code but virtually always in this one situation.  Hopefully this added information will help track down the bug.

Figure 1: default layout with minimized address field
![image](https://user-images.githubusercontent.com/44279601/65207705-884d6580-da60-11e9-8729-d4b4c7dcbee6.png)

Figure 2: expanded address field showing operand truncation
![image](https://user-images.githubusercontent.com/44279601/65208012-bed7b000-da61-11e9-875e-049ed5bd206c.png)
